src/tasks/LibraryBuilder/LibraryBuilder.cs

```diff
@@ -66,6 +66,26 @@ public bool IsSharedLibrary
         }
     }
 
+    /// <summary>
+    /// Determines whether or not the mono runtime auto initialization
+    /// tremplate, autoinit.c, is used.
+    /// </summary>
+    public bool UsesCustomRuntimeInitCallback { get; set; }
+
+    public string? AssetsPath { get; set; }
```

Comment:
`AssetsPath` seems too generic. Maybe we should rename this to AssemblyLocationPath?  I'm open to suggestions - I can get lost in naming with the best of them.

Reply:
Changed to `AssembliesLocation`